validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/exception/TagValueNotEqualsException.java

```diff
@@ -32,6 +32,6 @@ public TagValueNotEqualsException(String tagKey, String expected, String actual)
 
     @Override
     public String getCauseMessage() {
-        return String.format("[tag(%s) value]: expected=>{}, actual=>{%s}\n", tagKey, expected, actual);
+        return String.format("[tag(%s) value]: expected=>{%s}, actual=>{%s}%n", tagKey, expected, actual);
```

Comment:
I think you still need a `\n`.

Reply:
Yes. It misses %s only.

Reply:
These conversations totally confused me, to make it clear, I think what @wu-sheng mean is that it needs a \n or %n after `expected => {%s}`, not asking replacing the %n back to \n at the end of line, right?

Reply:
I just follow the existing codes, https://github.com/apache/skywalking-agent-test-tool/blob/3e3b3b91d09578dd1f05d6e7f7423959fc1720ba/validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/exception/ActualSegmentRefIsEmptyException.java#L31

they should be

```suggestion
        return String.format("[tag(%s) value]: expected=>{%s}, actual=>{%s}\n", tagKey, expected, actual);
```

Reply:
![image](https://user-images.githubusercontent.com/43675197/85215358-018c3700-b3aa-11ea-8aa8-fd2d43d20310.png)
I got suggestion by `SonarLint` and i think it's reasonable.

Reply:
That is a different thing, if you want to or should change all `\n` to `%n`, please consider another pull request.